plugins/pdr_deterministic_plugin/.ci/do_run_test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -x
+export SERVER_HOST=$SERVER_HOST
+expect << EOF
+spawn ssh admin@${SERVER_HOST}
+expect "Password:*"
+send -- "admin\r"
+expect "> "
+send -- "enable\r"
+expect "# "
+send -- "config terminal\r"
+expect "/(config/) # "
+send -- "ufm plugin pdr_deterministic enable\r"
+expect "/(config/) # "
+send -- "_shell"
```

Comment:
no need for the _shell 
if you login to the ufm with root it will get you to the shell

Reply:
didnt see that all good